tests/common/dualtor/mux_simulator_control.py

```diff
@@ -311,10 +311,26 @@ def toggle_all_simulator_ports_to_rand_selected_tor(mux_server_url, tbinfo, rand
 
     pytest_assert(_post(mux_server_url, data), "Failed to toggle all ports to {}".format(rand_one_dut_hostname))
 
+@pytest.fixture
+def toggle_all_simulator_ports_against_rand_selected_tor(mux_server_url, tbinfo, rand_one_dut_hostname):
```

Comment:
suggest renaming to `toggle_all_simulator_ports_to_rand_unselected_tor` for consistency with the `rand_unselected_dut` fixture we already have

Reply:
updated